### DISCUSSION TITLE
Some API changes on VWs end.

### DISCUSSION THREAD

**Discussion by MattiJarvinen** [tillsteinbach/VWsFriend#110] [General]
Unknown attribute readinessStatus with value {'connectionState': {'isOnline': True, 'isActive': False, 'batteryPowerLevel': 'comfort', 'dailyPowerBudgetAvailable': True}, 'connectionWarning': {'insufficientBatteryLevelWarning': False, 'dailyPowerBudgetWarning': False}}

Unknown attribute readinessBatterySupportStatus with value {'code': 4112, 'message': 'The capability for the requested operation is missing', 'group': 2, 'info': 'Internal error, please try again later. If the problem persists, please contact our support.'} in error section

**Comment by tillsteinbach** [comment]
Yes, these additions came yesterday. I think right now nothing worth storing. But I'm interested to see what it is good for in the next days. https://github.com/tillsteinbach/WeConnect-cli already got an update to support the new attributes. I plan to have a VWsFriend release today or tomorrow

**Reply by tillsteinbach** [reply]
New version 0.10 can deal with the changes
